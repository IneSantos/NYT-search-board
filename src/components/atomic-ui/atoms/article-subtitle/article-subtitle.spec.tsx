import React from "react";
import { render } from '@testing-library/react';
import ArticleSubtitle from "./article-subtitle";

describe("ArticleSubtitle atom tests", () => {
  test(`ArticleSubtitle component render`, () => {
    const { container } = render(
      <ArticleSubtitle subtitle="This is a subtitle"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
