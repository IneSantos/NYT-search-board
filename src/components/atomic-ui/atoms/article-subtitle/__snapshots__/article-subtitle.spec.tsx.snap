// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleSubtitle atom tests ArticleSubtitle component render 1`] = `
<div
  class="subtitle"
>
  This is a subtitle
</div>
`;
